src/web/src/components/Posts/Timeline.tsx

```diff
@@ -49,7 +50,7 @@ const Timeline = ({ pages, nextPage }: Props) => {
   // Iterate over all the pages (an array of arrays) and then convert all post
   // elements to <Post>
   const postsTimeline = pages.map((page: Post[]): Array<ReactElement> | ReactElement =>
-    page.map(({ id, url }: Post) => <PostComponent postUrl={url} key={id} />)
+    page.map(({ id }: Post) => <PostComponent postId={id} key={id} />)
```

Comment:
On Gitpod, the Post service returns localhost `postUrl` which is not reachable by the frontend. As a quick solution, I used `POST_URL/:id` to query individual posts. If we want to make the POST service return Gitpod URL for a post, I could also file a follow-up issue for this.